tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
@bgruening currently I'm doing this in the Jython scripts (jython_script.py) using the combination of these 2 things:

VALID_IMAGE_TYPES = [ ImagePlus.GRAY8 ]
...
    if image_type not in VALID_IMAGE_TYPES:
        # Convert the image to binary grayscale.
        IJ.run( input_image_plus_copy, "Make Binary","iterations=1 count=1 edm=Overwrite do=Nothing" )

Image metadata is something that is currently missing from the Galaxy framework.  Ideally, this information will be stored in metadata so that we can build tool param validators for images so that issues like this can be handled prior to tool execution.